src/libostree/ostree-gpg-verify-result.c

```diff
@@ -418,6 +415,14 @@ ostree_gpg_verify_result_get (OstreeGpgVerifyResult *result,
             child = g_variant_new_string (v_string);
             break;
 
+          case OSTREE_GPG_SIGNATURE_ATTR_FINGERPRINT_PRIMARY:
+            if (key != NULL && key->subkeys != NULL)
+              v_string = key->subkeys->fpr;
+            if (v_string == NULL)
+              v_string = "[unknown fingerprint]";
```

Comment:
This bit feels magical; how about we make this the empty string, and then clients can render that how they prefer (e.g. translated)?

Reply:
I was just aiming for consistency with _USER_NAME and _USER_EMAIL, which suffers the same problem - but I suppose that behaviour (of having a baked "empty string") is now ABI. I guess we can do better with the fingerprint on the basis it's more likely to be used as a lookup key, but arguably accessing/using any of these strings if key_missing is true is a programmer error anyway. I don't mind making the default "" if you prefer supporting this programmer error even if it's inconsistent with the others... :)